x/mint/client/cli/query.go

```diff
@@ -84,7 +85,7 @@ func GetCmdQueryInflation() *cobra.Command {
 				return err
 			}
 
-			return clientCtx.PrintOutput(res.Inflation)
+			return clientCtx.PrintString(fmt.Sprintf("%s\n", res.Inflation))
```

Comment:
why `PrintString`?

Reply:
`Inflation` is a `Dec` and we can't print simply a `Dec` with jsonpb, but a `Dec` can just be printed as a string.

The previous output was something like `"1234.00000"` in quotes. This actually just prints a number not in quotes `1234.00000"`. If we actually want to print a JSON string we will need to print something like `{"inflation":"1234.00000"}` wrapped in an object. Would that be preferable?

Reply:
Yes, `{"inflation":"1234.00000"}` would be self explanatory. May be return `clientCtx.PrintOutput(res)`?

Reply:
Yes, `{"inflation":"1234.00000"}` would be self explanatory. May be return `clientCtx.PrintOutput(res)`?